wolfcrypt/src/port/Espressif/esp_crt_bundle/README.md

```diff
@@ -0,0 +1,262 @@
+# wolfSSL Support for ESP-IDF Certificate Bundles
+
+These files are typically only used when integrating wolfSSL with the ESP-IDF
+and with the intention of using Certificate Bundles in the esp-tls component.
+
+See the ESP-IDF `idf.py menuconfig`. A recent version of the [wolfSSL Kconfig](https://github.com/wolfSSL/wolfssl/blob/master/IDE/Espressif/ESP-IDF/examples/template/components/wolfssl/Kconfig)
+file is needed. The [template example](https://github.com/wolfSSL/wolfssl/tree/master/IDE/Espressif/ESP-IDF/examples/template)
+can be use for creating a project-specific [wolfSSL component](https://github.com/wolfSSL/wolfssl/tree/master/IDE/Espressif/ESP-IDF/examples/template/components/wolfssl)
+when not using a [Managed Component](https://components.espressif.com/components/wolfssl/wolfssl).
+
+## Getting Started
+
+Use the `idf.py menuconfig`, 
```

Comment:
trailing whitespace on this line

Reply:
eek. I _usually_ don't need to worry about trailing whitespace. today I needed to edit my "ignore pattern". May have been an added feature in recent update in Visual Studio: 

![image](https://github.com/user-attachments/assets/58bc88cd-0f09-44d7-bfca-5af0018bb407)

Otherwise, I completely missed the long lines. Good catch! I've updated the code. There's one extra file this time around, oddly a space _missing_ between two words in a different `README.md` file.